qemu/tests/cfg/x86_cpu_model.cfg

```diff
@@ -2,9 +2,8 @@
     type = x86_cpu_model
     cpu_model_flags += ",enforce"
     auto_cpu_model = no
-    cpu_model = host
     kill_vm_on_error = yes
-    paused_after_start_vm = yes
+    start_vm = no
```

Comment:
pause vm  and query-cpu-definitions then cont vm looks fine, why changed it.

Reply:
The former process is to start vm,  query-cpu-definition, destory vm, then create vm with supported cpu model. Now we don't need to boot vm twice, just query cpu then start vm.